test/infrastructure/docker/api/v1beta1/devmachine_types.go

```diff
@@ -66,6 +66,119 @@ const (
 
+// DevMachine's v1Beta2 conditions that apply to all the supported backends.
+
+// DevMachine's Ready condition and corresponding reasons that will be used in v1Beta2 API version.
+const (
+	// DevMachineReadyV1Beta2Condition is true if the DevMachine's VirtualMachineProvisioned, NodeProvisioned,
+	// EtcdProvisioned (if present) and APIServerProvisioned (if present) conditions are true.
+	DevMachineReadyV1Beta2Condition = clusterv1.ReadyV1Beta2Condition
+
+	// DevMachineReadyV1Beta2Reason surfaces when the DevMachine readiness criteria is met.
+	DevMachineReadyV1Beta2Reason = clusterv1.ReadyV1Beta2Reason
+
+	// DevMachineNotReadyV1Beta2Reason surfaces when the DevMachine readiness criteria is not met.
+	DevMachineNotReadyV1Beta2Reason = clusterv1.NotReadyV1Beta2Reason
+
+	// DevMachineReadyUnknownV1Beta2Reason surfaces when at least one DevMachine readiness criteria is unknown
+	// and no DevMachine readiness criteria is not met.
+	DevMachineReadyUnknownV1Beta2Reason = clusterv1.ReadyUnknownV1Beta2Reason
+)
+
+// DevMachine's v1Beta2 conditions that apply to the in memory backend.
+
+// VirtualMachineProvisioned condition and corresponding reasons that will be used in v1Beta2 API version for a DevMachine's with in memory backend.
+const (
+	// DevMachineInMemoryVirtualMachineProvisionedV1Beta2Condition documents the status of the provisioning of a fake VirtualMachine for a DevMachine's with in memory backend.
+	DevMachineInMemoryVirtualMachineProvisionedV1Beta2Condition string = "VirtualMachineProvisioned"
+
+	// DevMachineInMemoryVirtualMachineWaitingForClusterInfrastructureV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend waiting for the cluster
+	// infrastructure to be ready.
+	DevMachineInMemoryVirtualMachineWaitingForClusterInfrastructureV1Beta2Reason = clusterv1.WaitingForClusterInfrastructureV1Beta2Reason
+
+	// DevMachineInMemoryVirtualMachineWaitingControlPlaneInitializedV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend waiting
+	// for the control plane to be initialized.
+	DevMachineInMemoryVirtualMachineWaitingControlPlaneInitializedV1Beta2Reason = clusterv1.WaitingControlPlaneInitializedV1Beta2Reason
+
+	// DevMachineInMemoryVirtualMachineWaitingForBootstrapDataV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend waiting for the bootstrap
+	// data to be ready.
+	DevMachineInMemoryVirtualMachineWaitingForBootstrapDataV1Beta2Reason = clusterv1.WaitingForBootstrapData1Beta2DataReason
+
+	// DevMachineInMemoryVirtualMachineWaitingForStartupTimeoutV1Beta2Reason documents the infrastructure for a fake VirtualMachine for a DevMachine's with in memory backend provisioning.
+	DevMachineInMemoryVirtualMachineWaitingForStartupTimeoutV1Beta2Reason = "WaitingForStartupTimeout"
+
+	// DevMachineInMemoryVirtualMachineProvisionedV1Beta2Reason documents a fake VirtualMachine for a DevMachine's with in memory backend being provisioned.
```

Comment:
This reason is used after it is provisioned not while it is being provisioned, right? (I think the comment expresses the latter atm)

Reply:
Fixed, PTAL